packages/core/theme-module/theme/components/LoginModal.vue

```diff
@@ -166,10 +212,11 @@ export default {
   },
   setup() {
     const form = ref({});
-    const isLogin = ref(false);
+    const isLogin = ref(1);
```

Comment:
I think it's better to use strings here (or enums) to make it more readable

Reply:
Done (string keys)